### PR TITLE
Fix line length in org.elasticsearch.routing

### PR DESCRIPTION
**Pull request by pgomulka**
Remove the line length suppression for this package and fix offending
lines

relates: #34884

### CODE REVIEW

**Comment by pgomulka** [comment]
retest this please

**Comment by pgomulka** [comment]
run the gradle build tests default-distro

**Comment by pgomulka** [comment]
retest this please

**Comment by pgomulka** [comment]
run default distro tests

File: server/src/test/java/org/elasticsearch/routing/AliasRoutingIT.java [hunk 1]
```diff
@@ -108,10 +113,10 @@ public void testAliasSearchRouting() throws Exception {
         createIndex("test");
         ensureGreen();
         assertAcked(admin().indices().prepareAliases()
-                .addAliasAction(AliasActions.add().index("test").alias("alias"))
-                .addAliasAction(AliasActions.add().index("test").alias("alias0").routing("0"))
-                .addAliasAction(AliasActions.add().index("test").alias("alias1").routing("1"))
-                .addAliasAction(AliasActions.add().index("test").alias("alias01").searchRouting("0,1")));
+                           .addAliasAction(AliasActions.add().index("test").alias("alias"))
```
Comment (ebadyano) [inline server/src/test/java/org/elasticsearch/routing/AliasRoutingIT.java]: The original indentation seems what is used throughout the rest of code. Does it make sense to use the original style vs indenting under the period of the line above?

Comment (pgomulka) [reply server/src/test/java/org/elasticsearch/routing/AliasRoutingIT.java]: I would prefer to change indentation for this file with fluent builder wrapping on dots.

Some of these chains are breaking the line length limit, I have reformatted them with intellij `Allign when multiline`. And then to keep this consistent across this file I formatted other fluent builders as well.

**Review by ebadyano — APPROVED** [review]
lgtm

**Pull request merged**